src/script/entity/Conversation.ts

```diff
@@ -185,6 +186,13 @@ export class Conversation {
     this.team_id = undefined;
     this.type = ko.observable();
 
+    /**
+     * If a conversation has the groupId property it means that it
+     * is MLS protocol based as this property is for MLS conversations only.
+     * @returns boolean
+     */
+    this.isUsingMLSProtocol = !!this.groupId;
```

Comment:
just curious where we are using this?

Reply:
changed, we're gonna use protocol instead but it's gonna be used for example in calling to differentiate how we send calling messages or adding people into conversations